MVPMarsOnboarding/MVPMarsOnboarding/Pages/CertificationModule.cs

```diff
@@ -10,14 +10,23 @@ namespace MVPMarsOnboarding.Pages
 {
     public class CertificationModule
     {
-        public string GetNewCertificateName(IWebDriver driver)
+        public IWebDriver driver;
+        CertificationModule CertificationModuleObj;
+        ProfilePage profilePageObj;
+
+        public CertificationModule(IWebDriver driver)
+        {
+            this.driver = driver;
+        }
+
+        public string GetNewCertificateName()
         {
             try
             {
                 //refresh the page
                 driver.Navigate().Refresh();
-                ProfilePage profilePageObj = new ProfilePage();
-                profilePageObj.GoToCertificationModule(driver);
+                profilePageObj = new ProfilePage(driver);
```

Comment:
should move to the profile page, also this profilePageObj = new ProfilePage(driver); should be done in the constructor,  

 public CertificationModule(IWebDriver driver)
        {
            this.driver = driver;

           // as an example but this should be present here
            this.profilePageObj = new ProfilePage(driver);
        }